x-pack/plugins/session_view/server/routes/io_events_route.ts

```diff
@@ -56,12 +57,8 @@ export const getTTYQueryPredicates = async (
 
   if (lastEventHits.length > 0) {
     const lastEvent: ProcessEvent = lastEventHits[0]._source as ProcessEvent;
-    const sessionEnded = lastEvent.event?.action === EventAction.end && lastEvent['@timestamp'];
     const lastEventTime = lastEvent['@timestamp'];
-    const rangeEnd =
-      sessionEnded && lastEventTime
-        ? parse(lastEventTime)?.add(30, 'second').toISOString()
-        : new Date().toISOString();
+    const rangeEnd = lastEventTime ? parse(lastEventTime).toISOString() : new Date().toISOString();
```

Comment:
this was fix an issue where events from a subsequent session were being pulled into a session.